post-installation/wpa_supplicant.txt

```diff
@@ -0,0 +1,92 @@
+WPA_SUPPLICANT
+________________________________________________________________________________
+
+wpa_supplicant is a cross-platform supplicant with support for WEP, WPA and WPA2
+(IEEE 802.11i). It is suitable for desktops, laptops and embedded systems. It is 
+the IEEE 802.1X/WPA component that is used in the client stations. It implements
+key negotiation with a WPA authenticator and it controls the roaming and IEEE 
+802.11 authentication/association of the wireless driver. [0]
+
+
+Configuration
+________________________________________________________________________________
+
+Begin by first verifying that you have wpa_supplicant installed:
+
+$ kiss b wpa_supplicant && kiss i wpa_supplicant
+
+At this point, you will want to create a wpa_supplicant file to store your 
+wireless network information and credentials:
+
+$ mkdir /etc/wpa_supplicant
+$ touch /etc/wpa_supplicant/wpa_supplicant.conf
+
+The following @/wpa_supplicant.conf can be used as a "starter" configuration
+file. Remember to replace the <BSSID> and <PASSWORD> with your actual wireless
+network credentials. For additional network requirements, refer to the 
+"wpa_supplicant" Arch Linux Wiki page [0].
+
+
+Generating a Passphrase
+________________________________________________________________________________
+
+NOTE: This section is not required but HIGHLY recommend since storing your
+      password in clear text is not a good practice. 
+
+To allow for quicker connections to a network whose BSSID is already known, we
+can make use of wpa_passphrase, a command line tool which generates the minimal
+configuration needed by wpa_supplicant: 
+
+$ wpa_passphrase <BSSID> <PASSWORD>
```

Comment:
`<BSSID>` (and all occurrences of `<word>`) will be interpreted as HTML by the browser and won't be displayed. You have two options here, using `&lt;`/`&gt;` or by swapping to a different character.  :+1: